build/scripts/PreCompileApp.ps1

```diff
@@ -53,6 +53,8 @@ if($appType -eq 'app')
                 else {
                     Compile-AppInBcContainer @tempParameters | Out-Null
                 }
+
+                $parameters.Value["preprocessorsymbols"] = Get-PreprocessorSymbols
```

Comment:
If the clean tags are set here, what's the point of _cleanModePreprocessorSymbols_ setting?

Reply:
None. They're not used. But AL-Go fails if we don't set anything. 

Maybe we should just write a warning [here](https://github.com/microsoft/AL-Go/blob/9dbee17f636ca9ba2649b45adc2b65f4fd66af84/Actions/RunPipeline/RunPipeline.ps1#L376) instead of throwing an error?

Reply:
This is a defensive check, so not sure about a warning. How would you fix such a warning?

But why automation PRs? 🤔 
My idea was just to add a check (in "Verify App Changes" maybe) that the `cleanModePreprocessorSymbols` are in order, based on the current version.
Then, to be on the safe side, `Get-PreprocessorSymbols` can dynamically add CLEAN preprocessor symbols for all versions between the current version and the version on _main_.

Generally speaking, CLEAN26 makes no sense in version 25. But so do all the clean builds we do all around, but _main_.